Readme.md

```diff
@@ -0,0 +1,46 @@
+# wedding project
+
+this project is all about online invitation.
+
+## App
+
+![Home](assets/images/home.png)
+![Wedding](assets/images/wedding.png)
+![jane-john](assets/images/jane-john.png)
+![rsvp](assets/images/rsvp.png)
+
+### Built With
+
+- HTML
+- CSS
+
+## Clone project
+
+- To get a local copy up and running follow these simple example steps.
+- Clone this repository with `git@github.com:kngcl/wedding.git` using your terminal.
+- Change to the project directory by entering: cd food in the terminal.
+
+## steps
+
+- $ git clone `git@github.com:kngcl/wedding.git`
+- $ `cd wedding`
+- $ `git checkout feature`
+
+## Start App
+
+- run `npm install`
+- run `npm start` in your CLI
+
+## Test
+
```

Comment:
You are not working on any test. section isn't required.

- [ ] remove test section